mm2src/coins/qrc20.rs

```diff
@@ -455,8 +458,11 @@ impl Qrc20Coin {
             .generate_qrc20_transaction(outputs)
             .await
             .mm_err(|e| e.into_withdraw_error(platform, decimals))
-            .map_err(|e| ERRL!("{}", e))?;
-        let _tx = try_s!(self.utxo.rpc_client.send_transaction(&signed).compat().await);
+            .map_err(|e| TransactionFutErr::Plain(ERRL!("{}", e)))?;
+        let _tx = match self.utxo.rpc_client.send_transaction(&signed).compat().await {
+            Ok(tx) => tx,
+            Err(err) => return TX_RECOVERABLE_ERR!(signed, "{:?}", err),
+        };
```

Comment:
> It can be simplified into
> 
> ```rust
> let _tx = try_tx_s!(self.utxo.rpc_client.send_transaction(&signed).compat().await);
> ```

This actually returns a transaction, but I also implemented returning tx error for `try_tx_s`. So I can do something like `try_tx_s!(self.utxo.rpc_client.send_transaction(&signed).compat().await, signed);`.

Reply:
Ooh yes, forgot to add the `signed` transaction. Good